app/views/admin/communities/analytics.haml

```diff
@@ -10,7 +10,11 @@
       %h3= t(".google_analytics_key")
     .row
       .col-6
-        = render :partial => "layouts/info_text", :locals => { :text => t(".google_analytics_key_info_text")}
+        - if APP_CONFIG.display_knowledge_base_articles
```

Comment:
We'd like to keep the logic of views in minimum, so instead of this, construct the instruction link and link text in controller and pass as locals. Then this whole if - else checking downgrades here to:

```ruby
= render :partial => "layouts/info_text", :locals { text: text, instructions_link: instructions_link }
```